features/remove_favorite.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Remove a product from favorites behavior
+
+  Background:
+
+         When I'm logged in with 1234
+
+  Scenario: As a valid user I remove 1 item from the favorites and I validate the removal of the item
+
+
+         Then I long press "Hose für Jungen blau"
```

Comment:
this test requires that the item is a favourite for it to adequately test. 

I'm not sure how Calabash decides on the test order - is it possible for this test to be ran before the add_favorite.feature test?  If there is a chance that it runs before, would this test fail on the first long press step?

Reply:
I tested the entire suite yesterday and this tests runs after the "add favorite"  one.  I could fit both tests in one feature to make sure that there is no issue with the order. Or I implement a relevant hook. What do you suggest?

Reply:
I'm going to comment on the "remove favorite" test that it needs to run after the "add favorite" one.